packages/uui-pagination/lib/uui-pagination.element.ts

```diff
@@ -62,6 +63,9 @@ export class UUIPaginationElement extends LitElement {
     super.connectedCallback();
     if (!this.hasAttribute('role')) this.setAttribute('role', 'navigation');
     this._visiblePages = this._generateVisiblePages(this.current);
+
+    demandCustomElement(this, 'uui-button');
+    demandCustomElement(this, 'uui-button-group');
```

Comment:
These are used directly in the render() method, so I think we should just be importing them directly. In fact it looks like we are already importing the button, so maybe just add the button-group?

Reply:
But can we ensure going forward that uui-button-group will make sure that uui-button is also registered as part of it?
Cause I think it would be good to be safe :-)

Reply:
Button group is already there at the top of this file, but yeah it is probably only being used as a type and is then swept away in production.

I guess what I'm saying is that if we import the components directly in this file, we don't need the demandCustomElement, but then we also potentially run into double registrations... :/

Reply:
Ah, I didn't look closely enough at the code to see that button was imported. I agree.

The button is the only one begin imported currently, and yes it is only used as a type so it would be good if we could know for sure that its there and begin registering. If we cant guarantee that then the Demands still has a purpose. If we can guarantee that they will be registered Then...

We should import the button-group, (button is already imported) and then we can get rid of the demands. :-)

Otherwise we need the demands to ensure that things are good.

Reply:
Yeah, let's just keep demandCustomElement. It doesn't hurt to have it at least.